src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100..900;1,100..900&family=Space+Grotesk:wght@300..700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="iMac picture"
+      />
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__raiting">
+        <div class="card__stars">
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+        </div>
```

Comment:
```suggestion
        <div class="card__stars card__stars--4">
          <span class="card__star"></span>
          <span class="card__star"></span>
          <span class="card__star"></span>
          <span class="card__star"></span>
          <span class="card__star"></span>
        </div>
```
Consider adapting the structure or just copying the structure from the stars task